lib/components/editor.js

```diff
@@ -180,6 +181,27 @@ class Editor extends React.Component {
   }
 
   /**
+   * When the editor receives a new 'state'
+   *
+   * @param {State} state
+   * @return {State} newState
+   */
+
+  onReceiveState = (state) => {
+    if (state == this.state.state) return
+
+    for (const plugin of this.state.plugins) {
+      if (!plugin.onReceiveState) continue
+      const newState = plugin.onReceiveState(state, this)
+      if (newState == null) continue
+      state = newState
+    }
+
+    if (!this.props.onReceiveState) return state
+    return this.props.onReceiveState(state)
```

Comment:
This call should be unnecessary because of the logic in `resolvePlugins` creating an "editor plugin", I believe.